malsim/sims/mal_simulator.py

```diff
@@ -293,6 +293,7 @@ def action_space(self, agent=None):
         return MultiDiscrete([num_actions, num_objects], dtype=np.int64)
 
     def reset(self, seed: Optional[int] = None, options: Optional[dict] = None):
+        # TODO: params not used by method
```

Comment:
I think the reset signature is defined by the PettingZoo or Gymnasium APIs, and we're just making it be compatible.

Reply:
That makes sense for the signature, but we probably don't need to send them to MalSimulator.reset() in AttackerEnv.reset() and DefenderEnv.reset then. Not sure though.

Reply:
Added TODO to Defender/AttackerEnv.reset instead